### ISSUE TITLE
VictoriaLogs web UI: window size when error

### ISSUE THREAD

**Issue by yuriydzobak** [VictoriaMetrics/VictoriaMetrics#7207]
### Describe the bug

Hi,  
When I write a long query and make a typo, the size of the window increases by horizontal, and it doesn't use the size of the window in the browser like it was before

### To Reproduce

1. open [play](https://play-vmlogs.victoriametrics.com/select/vmui)
2. write any long query and make typo in function
for example
```
error | unpack_json result_prefix ingress_ | stats by (ingress_x_forwarded_for,ingress_serviceName,ingress_httpRequest.requestMethod,dc,cluster,ingress_*) count() my_count | sort by (my_count) desc | my_count:>10
```
3. check the windows, it was increased by horizontaly

![screenshot-000478](https://github.com/user-attachments/assets/06762aa7-54b0-4fef-a399-4f9235c0536b)
when a long query with typo
![screenshot-000479](https://github.com/user-attachments/assets/fe68f11b-b24e-4ee1-afea-fd4f44ca5ad8)

### Version

0.33.0

**Comment by Loori-R** [comment]
Yes, I understand the reason; I will fix it shortly.

**Comment by valyala** [comment]
This bug should be fixed in [v0.36.0-victorialogs](https://docs.victoriametrics.com/victorialogs/changelog/). Closing the issue as fixed.

**Comment by f41gh7** [comment]
The bugfix was included into [v1.105.0](https://docs.victoriametrics.com/changelog/#v11050), [v1.102.5](https://docs.victoriametrics.com/changelog/#v11025) and [v1.97.10](https://docs.victoriametrics.com/changelog/#v19710)
